lib/h.js

```diff
@@ -7,7 +7,7 @@ module.exports = function h (tag, attr, children) {
   if (!Array.isArray(children)) children = [children]
   if (!attr) attr = {}
 
-  var open = '<' + tag.replace(/[\/!]$/, '')
+  var open = '<' + tag.replace(/[/!]$/, '')
```

Comment:
Is this because the escape on the `/` is redundant?

Reply:
yeah, this is from standard style